cli/tokens.go

```diff
@@ -67,14 +66,7 @@ func (r *RootCmd) createToken() *clibase.Cmd {
 				return xerrors.Errorf("create tokens: %w", err)
 			}
 
-			cliui.Infof(
-				inv.Stdout,
-				"Here is your token. 🪄\n\n",
-			)
-			cliui.Infof(inv.Stdout, cliui.Styles.Code.Render(strings.TrimSpace(res.Key))+"\n\n")
-			cliui.Infof(inv.Stdout,
-				"You can use this token by setting the --%s CLI flag, the %s environment variable, or the %q HTTP header.", varToken, envSessionToken, codersdk.SessionTokenHeader,
-			)
+			cliui.Infof(inv.Stdout, res.Key)
```

Comment:
Since we don't want any type of formatting, a `fmt.Fprint` might work here too. Infof could line-split (I think, but probably only if there are spaces).

Reply:
Ahh, I see! Will change.